torch/autograd/gradcheck.py

```diff
@@ -462,7 +462,10 @@ def check_undefined_grad_support(output_to_check):
                 return True
 
             # All backward functions must work properly if all output grads are undefined
-            outputs_to_check = [[torch._C._functions.UndefinedGrad()(o) for o in _differentiable_outputs(func(*tupled_inputs))]]
+            outputs_to_check = [[
```

Comment:
nit: Can you add an extra comment here:
"The isinstance check below allows to filter out Tensor-like objects that are returned by `_differentiable_outputs`."

Reply:
Added below.